pkg/sfu/forwarder.go

```diff
@@ -1984,20 +1984,6 @@ done:
 	if !targetLayer.IsValid() {
 		distance += (maxSeenLayer.Temporal + 1)
 	}
-	// TODO-REMOVE-AFTER-DEBUG
-	logger.Debugw(
-		"distance to desired",
-		"maxSeenLauer", maxSeenLayer,
-		"availableLayers", availableLayers,
-		"brs", brs,
-		"targetLayer", targetLayer,
-		"maxLayer", maxLayer,
-		"adjustedMaxLayer", adjustedMaxLayer,
-		"maxAvailableSpatial", maxAvailableSpatial,
-		"maxAvailableTemporal", maxAvailableTemporal,
-		"distance", distance,
-		"distanceToDesired", float64(distance)/float64(maxSeenLayer.Temporal+1),
-	)
```

Comment:
this I had added for a different reason. Saw one case of wrong distance. But, have scanned hundreds of these logs and cannot spot any errors. Still puzzled how that one instance had incorrect distance.